bin/main

```diff
@@ -1,3 +1,125 @@
 #!/usr/bin/env ruby
 
-puts "hello word"
+require_relative '../lib/board'
+
+class UserInterface
+  attr_accessor :player_one, :player_two, :game
+
+  def initialize
+    @arr = []
+    @game = Board.new
+    @player_one = ''
+    @player_two = ''
+  end
+
+  def num_check(move)
+    if @arr.include?(move)
+      puts 'Enter a Valid number thats not already played'
+      false
+    elsif !(1..9).include? move
+      puts 'Enter a number between 1 and 9'
+      false
+    else
```

Comment:
- [ ] [Optional] You have a game logic in your main.rb file. 
All game-logic related resources (classes and/or modules) are kept in the `lib` directory. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

The `main` files should be responsible for displaying and getting inputs. In this case, the array that keeps hold of the moves should be in the `lib` directory so that the decision comes from the logic file.